examples/fungible-token/tests/general.rs

```diff
@@ -0,0 +1,49 @@
+extern crate env_logger;
+// #[war]
+#[allow(unused_imports)]
+#[macro_use]
+extern crate log;
+extern crate quickcheck;
+#[macro_use(quickcheck)]
+extern crate quickcheck_macros;
+mod utils;
+
+use near_primitives::types::Balance;
+use utils::init_contract;
+
+pub fn ntoy(near_amount: Balance) -> Balance {
+    near_amount * 10u128.pow(24)
+}
+
+#[quickcheck]
+fn qc_test_total_supply(initial_supply: Balance) -> bool {
+    let deposit_amount = ntoy(initial_supply);
+    let (mut runtime, root) = init_contract(deposit_amount);
+    let total_supply = root.get_total_supply(&mut runtime);
+    assert_eq!(total_supply, deposit_amount);
+    assert_eq!(
+        root.get_balance(&mut runtime, root.account_id().into()),
+        deposit_amount
+    );
+    return true;
+}
+
+#[quickcheck]
+fn qc_test_transfer(initial_supply: Balance) -> bool {
+    let deposit_amount = ntoy(initial_supply);
```

Comment:
How quickcheck generates values? You're doing `let deposit_amount = ntoy(initial_supply);` for some reason that's why I'm asking..

Reply:
It seems to start low. You can add `info!("{}",..)` and the `RUST_LOG=info cargo test`

Reply:
But why it can't overflow? I mean `ntoy(initial_supply)`

Reply:
Btw, you can do `cargo test -- --nocapture` to see output of `println!` or `dbg!()` for debug. No need for RUST_LOG..

Reply:
Ah good to know.  The other reason for using info was because that's what quickcheck uses and you can get more info from it.